src/coq/decompiler/decompiler.ml

```diff
@@ -233,19 +288,15 @@ let rec semicolons sigma (t : tactical) : tactical =
 
 (* Try implicit arguments to rewrite functions. *)
 let rec rewrite_implicit sigma (t : tactical) : tactical =
```

Comment:
I didn't catch this earlier, but when `run_tac` fails, this needs error handling, I think by surrounding the whole thing with a `try ... with _ -> t`?